### DISCUSSION TITLE
Bootstrap imported in a NextJS Project using script tag

### DISCUSSION THREAD

**Discussion by khurramraheel** [twbs/bootstrap#40794] [Q&A]
Bootstrap imported in a NextJS Project using script tag, what could be the issue.

**Comment by ynng3** [comment]
In React projects like Next.js, instead of directly importing JavaScript script files, the module called react-bootstrap is used.

```sh
npm install react-bootstrap
```
## Reference
[React Bootstrap](https://react-bootstrap.netlify.app/)
